corehq/dbaccessors/couchapps/all_docs.py

```diff
@@ -0,0 +1,17 @@
+from corehq.preindex import get_preindex_plugin
+
+
+def _get_all_docs_dbs():
+    return get_preindex_plugin('couchapps').get_dbs('all_docs')
```

Comment:
Resolved in commit below. Also added a docstring to the function pointing out the dependency on this config